### ISSUE TITLE
Sizes of logo between episodes 8 and 9 are inconsistent

### ISSUE THREAD

**Issue by tsimonq2** [alexmurray/ubuntu-security-podcast#2]
![image](https://user-images.githubusercontent.com/3689821/47697541-84f78400-dbd9-11e8-85dc-a44d73ee28d1.png)

The image explains it all.

**Comment by alexmurray** [comment]
Yeah that is something to do with the theme differentiating between current episode vs older episodes - which seems kinda backwards since ideally current ep would be larger or at least the same size...
